### DISCUSSION TITLE
Language Level (A1, B2) filtering

### DISCUSSION THREAD

**Discussion by kotyzap** [met/subadub#19] [Ideas]
What about instead of "hiding" the end of sentences filter only the words based on some commonly used language levels (A1, B2, C1, IELTS5). Vocabulary lists are generally available such as B3 English level - https://www.vocabulary.com/lists/187041 

P.S. Byla by to pecka filtrovat slova dle jazykových úrovní. Navíc by to šlo aplikovat i na jiné jazyky... 

Level B2 je cca 1000 slovíček... kolik to je kB .csv souboru? Pár kilo? 
![Screen Shot 2021-01-09 at 15 14 20](https://user-images.githubusercontent.com/21118976/104093896-e45b7f00-528d-11eb-8bf2-05b8778c16a7.png)

**Comment by met** [comment]
I can use only word lists with some open licence (Creative Commons, MIT etc.)
I have already found stop-words lists for many languages with open licence (no problem here). And I am experimenting with them. But these lists have only around hundred of most basic words in every language.

I still need to find some large vocabulary list with good licence.
